File: Intro/java/isPrime.java
class Source {
  public static boolean isPrime(int number) {
    if (number < 2) {
      return false;
    }
    for (int i = 2; i <= Math.sqrt(number); i += 1) {
      if (number % i == 0) {
        return false;
      }
    }
    return true;
  }

  public static void run() {
    // this function behaves as `main()` for the 'run' command
    // you may sandbox in this function, but should not remove it
  }
}

File: Intro/go/isprime.go
package main

import (
	"fmt"
)

func isPrime(n int) bool {
	if n < 2 {
		return false
	}
	for i := 2; i*i <= n; i++ {
		if n%i == 0 {
			return false
		}
	}
	return true
}

func main() {
	fmt.Println(isPrime(2))   // true
	fmt.Println(isPrime(17))  // true
	fmt.Println(isPrime(18))  // false
	fmt.Println(isPrime(-5))  // false
}